### ISSUE TITLE
HTML formatting in field values

### ISSUE THREAD

**Issue by spficklin** [tripal/tripal#1891]
### Task Description

Recently, a PR changed the `title_format` of an organism entity to have HTML tags in the title. This allows the organism to have an italicized genus and species.    The problem I see with this is that it puts formatting on a value and assumes that the visualization of that formatting should be HTML.   This can be problematic with web services because the values that people can pull from the service should not force any type of formatting. The user should be able to use those values however they want.  

I think it's important that we don't force formatting on data values.  This is why we have a separation between the field type class and the field formatter class.  All formatting happens in the formatter.   I'm not sure how to fix this though.   We may be able to override the title field class but the token substitutions for the title happens at the publishing step, so if we then want to apply HTML formatting we'd have to know how to reapply the title token string with HTML tags. 

Doug suggested that for web services we strip out the HTML elements, which could work, but then what if someone still wants some type of formatting for their end use?  They could either rebuild the title using the other fields or have to figure out how to apply the italics at the right place.   

Here's a suggestion.  In the entity YML file add a `title_plain` element that has not formatting, and insted of HTML in the `title_format` we allow markdown.  The markdown can be coverted to whatever format someone wants and is more general.

### Branch Name

tv4g[0-9]-issue\d+-[optional short descriptor]

**Comment by spficklin** [comment]
Another point rasied by Doug:  Field values we save via the GUI may have formatting if we use the CKEditor, so we'll have the same problem.

**Comment by dsenalik** [comment]
Though this is a tiny subset of HTML that we are talking about &lt;em&gt; and &lt;strong&gt; and maybe a few others - converting to another markup format would not be too difficult 🤞

**Comment by spficklin** [comment]
Here's another issue:

```
<em>Gorilla gorilla</em> <em></em>
```

Extra `<em>` when no values for tokens. It doesn't show up on the webpage but will look funny otherwise.

**Comment by spficklin** [comment]
> Though this is a tiny subset of HTML that we are talking about `<em>` and `<strong>` and maybe a few others - converting to another markup format would not be too difficult 

I agree, those are easy enough to find and markdown is just another formatting too.  I just don't like HTML as a "generic" way to define formatting and prefer something more simple and human-readable.
